python/dgl/graphbolt/itemset.py

```diff
@@ -105,6 +135,8 @@ def __iter__(self) -> Iterator:
                 yield tuple(item)
 
     def __len__(self) -> int:
+        if isinstance(self._items, int):
+            return self._items  # TODO: discuss the return value here.
```

Comment:
What are the other options?

Reply:
I'm just not sure whether this is correct.